.gitignore

```diff
@@ -60,7 +60,6 @@ gradle-org.wycliffeassociates.otter.jvm.app.setting
 *.war
 *.nar
 *.ear
-*.zip
```

Comment:
don't remove this, if you need to upload a zip to commit for a test, you should be able to do that manually; I'd prefer arbitrary accidental zip files not get picked up as changes and accidentally committed